client/task-list/tasks/payments/RemotePayments/components/PaymentConnect.js

```diff
@@ -151,39 +113,61 @@ export const PaymentConnect = ( {
 		/>
 	);
 
-	if ( state === 'error' ) {
```

Comment:
It makes sense to lose this here since we've moved the fetch out of this component, but I _have_ wondered in general about our approach to error handling in the app. No response needed, just wondering "aloud".